spec/transactions_spec.rb

```diff
@@ -0,0 +1,29 @@
+describe Transactions do
+
+  let(:transactions) { Transactions.new }
+  let(:money_tracker) { MoneyTracker.new }
```

Comment:
It's not quite clear why `MoneyTracker` is being instantiated in order to test `Transactions`